oximeter/oximeter/schema/switch-data-link.toml

```diff
@@ -251,6 +252,26 @@ description = "Serial number of the sled managing the link's switch"
 type = "uuid"
 description = "ID of the switch the link is on"
 
+[fields.switch_fab]
+type = "string"
+description = "Fabric the switch the link is on"
```

Comment:
I think it was just an assumption on my part, but I thought this referred to the fabrication factory in which the ASIC was made. Could we actually confirm that somewhere? I can't find any clarification in the internal documentation Intel has provided us, but I am probably just not searching effectively.